Makefile

```diff
@@ -116,11 +116,23 @@ generate: $(CONTROLLER_GEN) ## Generate code
 
 .PHONY: generate-deepcopy
 generate-deepcopy: $(CONTROLLER_GEN) ## Generate deepcopy files
-	$(CONTROLLER_GEN) object:headerFile=./hack/boilerplate/boilerplate.generatego.txt paths=./api/...
+	$(CONTROLLER_GEN) \
+		object:headerFile=./hack/boilerplate/boilerplate.generatego.txt \
+		paths=./api/... \
+		crd:preserveUnknownFields=false \
+		output:crd:dir=./config/crd/bases
 
 .PHONY: generate-manifests
 generate-manifests: $(CONTROLLER_GEN) ## Generate manifests e.g. CRD, RBAC etc
-	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:dir=$(CRD_ROOT) output:webhook:dir=$(WEBHOOK_ROOT) output:rbac:dir=$(RBAC_ROOT)
+	$(CONTROLLER_GEN) \
+		$(CRD_OPTIONS) \
+		rbac:roleName=manager-role \
+		webhook \
+		paths="./..." \
+		crd:preserveUnknownFields=false \
```

Comment:
This should probably be changed up top where `CRD_OPTIONS` is defined, also unsetting the trivialversions flag that is currently set.

Reply:
added the preserveUnknownFields option to `CRD_OPTIONS`